config.sample.yaml

```diff
@@ -51,6 +51,7 @@ transform:  # Optional
       local_disk_min_size: 600
       category: "Memory Optimised"
     autotermination_minutes: 120
+    runtime_engine: STANDARD  # Optional: STANDARD or PHOTON
```

Comment:
might be good to have somewhere in the documentation to the benefits and coverage of PHOTON?

Reply:
Good shout, have added a link to the Photon docs